src/raw/mod.rs

```diff
@@ -855,60 +852,49 @@ impl<T, A: Allocator + Clone> RawTable<T, A> {
 
     /// Attempts to get mutable references to `N` entries in the table at once.
     ///
-    /// Returns an array of length `N` with the results of each query. For soundness,
-    /// at most one mutable reference will be returned to any entry. An
-    /// `Err(UnavailableMutError::Duplicate(i))` in the returned array indicates that a suitable
-    /// entry exists, but a mutable reference to it already occurs at index `i` in the returned
-    /// array.
+    /// Returns an array of length `N` with the results of each query.
+    ///
+    /// When `validate_hashes_are_unique` is true, at most one mutable reference will be returned to
+    /// any entry. `None` will be returned if any of the hashes are duplicates.
+    ///
+    /// `None` will be returned if the hash is not found.
     ///
     /// The `eq` argument should be a closure such that `eq(i, k)` returns true if `k` is equal to
     /// the `i`th key to be looked up.
-    ///
-    /// This method is available only if the `nightly` feature is enabled.
-    #[cfg(feature = "nightly")]
-    pub fn get_each_mut<const N: usize>(
+    pub fn get_many_mut<const N: usize>(
         &mut self,
         hashes: [u64; N],
+        validate_hashes_are_unique: bool,
```

Comment:
This needs to be an unsafe function if validation can be disabled. I would prefer if there was a separate `unchecked` function which is unsafe.

Reply:
Updated